CHANGELOG.md

```diff
@@ -1,3 +1,9 @@
+## [v2.7.0](http://github.com/ndustrialio/contxt-sdk-js/tree/v2.7.0) (2020-01-27)
```

Comment:
Depending on whether or not this gets approved/merges/published today this date _may_ need to be updated. Just be aware